sys-fs/bcachefs-tools/bcachefs-tools-9999.ebuild

```diff
@@ -59,15 +140,14 @@ BDEPEND="
 		)
 	')
 	$(unpacker_src_uri_depends)
-	<sys-devel/clang-$((${LLVM_MAX_SLOT} + 1))
+	$(llvm_gen_dep '
+		sys-devel/clang:${LLVM_SLOT}
+		sys-devel/llvm:${LLVM_SLOT}
```

Comment:
Sorry, one more thing. Are you sure LLVM is needed? Isn't it just clang for libclang for bindgen?

Reply:
I'm honestly not sure since I'm not familiar at all with bindgen, @Kangie would know that most likely.